README.md

```diff
@@ -110,40 +112,35 @@ The following setup creates a ring compliant handler.
       (s1st/executor)))
 ```
 
-## Complete Example Projects
+### Commands for Development
 
-Checkout the Leiningen templates as mentioned in the Kickstart section for working examples.
-
-### Friboo
+```shell
+# get the source
+$ git clone https://github.com/zalando/swagger1st.git
+$ cd swagger1st
 
-[Friboo](https://github.com/zalando-stups/friboo) is [Zalando](http://tech.zalando.com/)'s opinionated Clojure
-microservice library which uses swagger1st at its base for RESTful HTTP endpoints and also integrates with the
-[component](https://github.com/stuartsierra/component) framework. See the following projects who are real world
-applications of Zalando's cloud infrastructure based on Friboo with swagger1st:
+# run the tests
+$ lein test
 
-* [Kio](https://github.com/zalando-stups/kio)
-* [PierOne](https://github.com/zalando-stups/pierone) (a complete Docker registry based on S3)
-* [essentials](https://github.com/zalando-stups/essentials)
-* [TWINTIP](https://github.com/zalando-stups/twintip-storage)
-* [mint](https://github.com/zalando-stups/mint-storage)
+# run all tests, including performance benchmarks
+$ lein test :all
 
-### swagger-mock
+# build an own artifact for local development
+$ lein install
 
-[swagger-mock](https://github.com/zalando/swagger-mock) is an example how to also use swagger1st. Since you have the
-complete data of your Swagger definition at hand, you can modify it before processing and also use its definition
-information during execution. This allows the swagger-mock to hook into your definition and parse the examples from it
-in order to return them on each request.
+# release a new version
+$ lein release :minor
+```
 
-## Compatibility Overview
+For interactive development, you can start a REPL by typing `lein repl`.
 
-swagger1st aims to implement all features of the Swagger 2.0 specification. Everything that you can define with Swagger
-should be handled by swagger1st, so that you only have to write your business logic. Version 1.0 will implement all
-elements (that makes sense to handle). Until then, the following document shows the current supported aspects of the
-specification:
+### Projects Using Swagger1st in Production
```

Comment:
I'm using a standardized capitalization format for section titles (capitalize all words +1 syllables) so it's capitalized to conform